cypress/integration/movie_list_spec.js

```diff
@@ -0,0 +1,119 @@
+describe('Movie List', function() {
+  beforeEach(function() {
+    cy.visit('/')
+  })
+
+  context('12 Angry Men', function() {
+    beforeEach(function() {
+      cy.get('#movie-list .movie-row:first').as('firstMovie')
```

Comment:
Having aliases created in the `beforeEach` is a nice pattern. Similar to having `const` declared at the top of a function.